src/main/java/mServer/crawler/sender/arte/MediathekArte_de.java

```diff
@@ -35,15 +35,14 @@
 import de.mediathekview.mlib.tool.Log;
 import java.util.ArrayList;
 import java.util.Collection;
-import java.util.List;
 import java.util.concurrent.CopyOnWriteArrayList;
 import mServer.crawler.CrawlerTool;
 import mServer.crawler.FilmeSuchen;
 import mServer.crawler.sender.MediathekReader;
 
-public class MediathekArte_de extends MediathekReader
-{
-    /*
+public class MediathekArte_de extends MediathekReader {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this class name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00101)